### DISCUSSION TITLE
Error : EFI Floppy... No compatible bootloader found.

### DISCUSSION THREAD

**Discussion by Shruthishettys** [vmware/packer-examples-for-vsphere#849] [Help] [answered]
Could you please assist me in understanding whether it is essential to utilize 'cd_files' for uploading the Windows ISO and 'cd_content' for the answer file?
In my scenario, I am using 'floppy_files' and 'floppy_content' as follows, as 'cd_files' necessitates the installation of a command-line tool for managing ISO creation.
However, when I execute this, it generates an error stating 'EFI Floppy... No compatible bootloader found.' Could you provide guidance on this issue?"

Code reference:
iso_paths = ["[${var.common_iso_datastore}] ${var.iso_path}/${var.win11_iso_file}", "[${var.common_iso_datastore}] ${var.iso_path}/${var.vmware_iso_file}"]
boot_command = [""]
floppy_files = ["windows-init.ps1","windows-vmtools.ps1"]
floppy_content = {
"autounattend.xml" = templatefile("autounattend.pkrtpl.hcl", {
winrm_password = var.winrm_password
})
}

**Comment by tenthirtyam** [comment, marked as answer]
It's not required from the plug-in perspective, but it is for **this** project. 

If you wish to use the floppy method you would might need to make a modification to the drive used for VMware Tools (I already have a code enhancement to make this more dynamic).

But it should work for the installation portion.

Sorry, it's not something that I'll be adding to the project as it's standardized on the http or cd method for linux and cd for windows.

**Comment by tenthirtyam** [comment]
What's the concerns with the addition of the ISO utility?

**Reply by Shruthishettys** [reply]
I actually use Jenkins pipelines to run the packer commands. And the Jenkins agents are managed by other team, so installation of cmd line tool for ISO becomes a bit challenging in my environment.
So, I would prefer to use Floppy but then it throws error

**Comment by neopsyon** [comment]
@tenthirtyam - I am having issues with loading autounattend.xml with Windows 11. My boot configuration is set to EFI, and the Floppy seems not to be supported via EFI. How do I load the autounattend.xml file?

**Reply by tenthirtyam** [reply]
This project uses EFI with secure boot and cd_content. I assume this has been modified from the original as well?
